apt/private/deb_translate_lock.bzl

```diff
@@ -107,6 +93,76 @@ filegroup(
 )
 """
 
+def _package_build(template, package, repo_name):
```

Comment:
can you revert this change? I'd like to see the actual diff rather than the refactor.

Reply:
Ah! I misunderstood your comments, I thought the issue was with the rest of the code from the rebase that made it hard to review!

I'll make another proposal.